pkg/extensions/tbtc/recovery/storage_test.go

```diff
@@ -0,0 +1,57 @@
+package recovery
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestDerivationIndexStorage_Read(t *testing.T) {
+	dir, err := ioutil.TempDir("", "example")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+	dis, err := NewDerivationIndexStorage(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+	index, err := dis.Read("extendedPublicKey")
+	if err != nil {
+		t.Fatal(err)
+	}
+	expectedIndex := 0
+	if index != expectedIndex {
+		t.Errorf("incorrect extendedPublicKey index\nexpected: %d\nactual:   %d", expectedIndex, index)
+	}
+}
+
+func TestDerivationIndexStorage_WriteThenRead(t *testing.T) {
```

Comment:
Shouldn't it be `SaveThenRead` given that we `.Save(...)`?

Reply:
yup!